From 57360b1700e298a22ac86debe0fcf52064b91a50 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:31:10 -0400
Subject: [alsa] core: Fix format string mismatch in seq_midi.c

Message-id: <140972823573-279-patchset-jkysela@redhat.com>
Patchwork-id: 90975
O-Subject: [RHEL 7.1 PATCH 278/442] ALSA: core: Fix format string mismatch in seq_midi.c
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Fix format string mismatch in snd_seq_midisynth_register_port().
Argument type of p is unsigned int.

Signed-off-by: Masanari Iida <standby24x7@gmail.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: 53403a8013c5523c400987bf43a8c43963127dda)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/core/seq/seq_midi.c b/sound/core/seq/seq_midi.c
index 3e05c55..a1fd77a 100644
--- a/sound/core/seq/seq_midi.c
+++ b/sound/core/seq/seq_midi.c
@@ -362,13 +362,13 @@ snd_seq_midisynth_register_port(struct snd_seq_device *dev)
   if (! port->name[0]) {
    if (info->name[0]) {
     if (ports > 1)
-     snprintf(port->name, sizeof(port->name), "%s-%d", info->name, p);
+     snprintf(port->name, sizeof(port->name), "%s-%u", info->name, p);
     else
      snprintf(port->name, sizeof(port->name), "%s", info->name);
    } else {
     /* last resort */
     if (ports > 1)
-     sprintf(port->name, "MIDI %d-%d-%d", card->number, device, p);
+     sprintf(port->name, "MIDI %d-%d-%u", card->number, device, p);
     else
      sprintf(port->name, "MIDI %d-%d", card->number, device);
    }
-- 
1.7.1